opam

```diff
@@ -25,7 +25,7 @@ depends: [
     "oasis" {build & >= "0.4.0"}
     "base-unix"
     "base-threads"
-    "core_kernel"
+    "core_kernel" { >= "0.9.0"}
```

Comment:
it should be `v0.9`, notice the leading `v`.

Reply:
i was wondering that and was going to let travis tell me, but this is quicker. (It did build on my system).

Reply:
I'll change that after the above is answered re: `Chan`.